django_add_default_value/add_default_value.py

```diff
@@ -159,8 +161,18 @@ def deconstruct(self):
             {"model_name": self.model_name, "name": self.name, "value": self.value},
         )
 
-    def initialize_vendor_state(self, connection):
-        self.set_quotes(connection.vendor)
+    def initialize_vendor_state(self, schema_editor):
+        import django
```

Comment:
Any specific reason why this is a local import? I guess it could also be at top-level